### PR TITLE
Updates to EIP-1985

### LINKED ISSUE #92: Lower maximum transaction gas limit
## Overwiew

Maximum allowed transaction gas limit is 2<sup>63</sup> - 1.
## Specification

Any transaction with the gas limit greater than 2<sup>63</sup> - 1 should be ignored by a client.
## Motivation

Designing Ethereum VM for unrealistic case where amount of gas is greater than 2<sup>64</sup> is a waste of time.

Allowing the gas counter to fit regular CPU register on 64-bit targets is a big performance win. One more bit is reserved to allow faster overflow detection.

### PR DESCRIPTION
**Pull request by axic**
Closes #92.

### CODE REVIEW

**Comment by chfast** [comment]
The `PC` is also limited by the chain code size limit.

**Comment by axic** [comment]
@chfast added more limits.

Would like to also include opcodes which take these arguments as inputs, which have to be limited, e.g. CALL.

File: EIPS/eip-1985.md [hunk 1]
```diff
@@ -50,15 +51,22 @@ They restrict the results (i.e. values pushed to the stack) of the instructions
...
-   - `MSIZE` (`0x59`).
+   - `MSIZE` (`0x59`),
+   - `PC` (`0x58`).
+
+They restrict the acceptable input value ranges (as defined above) of the instructions listed below.
+If a value outside of this ranges are encountered, the execution stops with an exceptional halt consuming all remaining gas.
+
+1. CALL* / CREATE*
 
+2. CALLDATACOPY / CODECOPY / EXTCODECOPY / RETURNDATACOPY / CALLDATALOAD / MLOAD / MSTORE / MSTORE8 / REVERT / RETURN
```
Comment (axic) [inline EIPS/eip-1985.md]: @chfast any idea how to write this section nicely?

File: EIPS/eip-1812.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 eip: 1812
 title: Ethereum Verifiable Claims
-author: Pelle Braendgaard <@pelle>
+author: Pelle Braendgaard (@pelle)
```
Comment (axic) [inline EIPS/eip-1812.md]: I'll pull this into another PR, it is fixing a typo.

File: EIPS/eip-1985.md (hunk 1 above)
Comment (axic) [reply EIPS/eip-1985.md]: @chfast ping

Comment (chfast) [reply EIPS/eip-1985.md]: Have some ideas, not little time at the moment.

Comment (axic) [reply EIPS/eip-1985.md]: Moved this commit to #2276.

**Pull request merged**